script/.contributors_aliases.json

```
@@ -354,7 +354,7 @@
     "name": "Jacob Bogdanov"
   },
   "jacobtylerwalls@gmail.com": {
-    "mails": ["jacobtylerwalls@gmail.com"],
+    "mails": ["jacobtylerwalls@gmail.com", "jwalls@azavea.com"],
```

I signed one commit in error, so I would prefer not to memorialize my mistake :-)
```suggestion
    "mails": ["jacobtylerwalls@gmail.com"],
```

---

GitHub didn't say which email would be used when "squashing and merging" a PR. I assumed it would be the email on the first commit, not the ones being squashed away, which was signed with the wrong email. I was wrong.

---

Ah, well, we can't do anything about this, then, because we'd have to record not to record this to avoid manually updating every time. Rats.

---

Yeah, this is what 'recording not to record' look like unfortunately :sweat_smile:. I've made the mistakes too (when using my employer computer on my free time with git's default configuration for a cloned repo). Once it's in the history...